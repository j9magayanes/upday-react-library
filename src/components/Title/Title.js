import React from 'react';
import './title.css';


export const Title = ({ args }) => {

  return (
   <h1 className={args.appName==='meinestadt'?'title meinestadt':'title'}>{args.title}</h1>
  );
};
