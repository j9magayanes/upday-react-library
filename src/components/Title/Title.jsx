import React from 'react';
import './title.css';


export const Title= ({args}) => {
        return (
            <div className="page">
            <article>
            <div className="padded-lr legal">
                <h1 id="imprint">{args.title}</h1>
            </div>
            </article>
            </div>)}
