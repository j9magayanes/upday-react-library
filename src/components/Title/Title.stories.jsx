import React from 'react';

import { Title } from './Title';


export default {
  title: 'Form/Title',
  component: Title,
};

const Template = () =><Title args={{title: "Title"}}  />;


export const Default = Template.bind({});