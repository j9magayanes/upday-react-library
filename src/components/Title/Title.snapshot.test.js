// Title.test.js

import React from 'react';
import renderer from 'react-test-renderer';
import { Title } from './Title';

test('renders Title component correctly', () => {
  const args = {
    appName: 'meinestadt',
    title: 'Your Title Here',
  };

  const component = renderer.create(<Title args={args} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});

