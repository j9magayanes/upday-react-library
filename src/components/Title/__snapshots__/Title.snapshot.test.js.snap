// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title component correctly 1`] = `
<h1
  className="title meinestadt"
>
  Your Title Here
</h1>
`;
